packages/react-composites/src/composites/ChatComposite/FileDownloadErrorBar.tsx

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { IIconProps, MessageBar, MessageBarType, Stack } from '@fluentui/react';
+import React from 'react';
+
+/**
+ * @private
+ */
+export interface FileDownloadErrorBarProps {
+  /**callback to dismiss the download error message */
+  onDismissDownloadErrorMessage?: () => void;
+  /** Error message to render */
+  fileDownloadErrorMessage?: string;
+}
+
+/**
+ * @private
+ */
+export const FileDownloadErrorBar = (props: FileDownloadErrorBarProps): JSX.Element => {
+  const { fileDownloadErrorMessage, onDismissDownloadErrorMessage } = props;
+  const messageBarIconProps = (): IIconProps | undefined => {
```

Comment:
Why is this is a callback (It's called on line 34). This can just be a file-private constant since the value never changes.

Reply:
done